site/en/reference/cli/concepts/netdata.md

```diff
@@ -125,6 +141,15 @@ The Publisher requires `OPENTHREAD_CONFIG_NETDATA_PUBLISHER_ENABLE`.
     `otServerConfig::mStable`. The RLOC is also appended to the end of the
     record.
 
+1.  Display the current length, in number of bytes, of Partition's Thread Network
```

Comment:
Need to determine if this is useful to have here or not.  When would it be useful to know the netdata length?  Seems like you would want to use this with maxlength to monitor the size of the network data and then do something if it's too big?

Reply:
This is again useful for debugging/diagnostic.

A user can get the Network Data as raw bytes using `netdata show -x` and count the number of bytes, but using `netdata length` make it easier :)